prinz_h2o/src/main/scala/pl/touk/nussknacker/prinz/h2o/repository/H2OModelPayload.scala

```diff
@@ -0,0 +1,6 @@
+package pl.touk.nussknacker.prinz.h2o.repository
+
+import java.io.InputStream
+
+// Tip: MOJO readers also support java.net.URL in place of InputStream
+class H2OModelPayload(var inputStream: InputStream, var name: String, var version: String)
```

Comment:
case modifier was good here
 case class + object with apply function was for me without any sense